templates/scenarios/js/notification-restify/teamsfx/app.local.yml

```diff
@@ -7,6 +7,22 @@ registerApp:
     # Output: following environment variable will be persisted in current environment's .env file.
     # TEAMS_APP_ID: the id of Teams app
 
+provision:
+  - uses: botAadApp/create # Creates a new AAD app for bot if BOT_ID environment variable is empty
+    with:
+      name: notification-bot
+    # Output: following environment variable will be persisted in current environment's .env file.
+    # BOT_ID: the AAD app client id created for bot
+    # BOT_PASSWORD: the AAD app client secret created for bot
+
+  - uses: m365Bot/createOrUpdate # Create or update the bot registration on dev.botframework.com
+    with:
+      botId: ${{BOT_ID}}
+      name: notification-bot
+      messagingEndpoint: ${{TUNNEL_ENDPOINT}}/api/messages
+      description: ""
+      iconUrl: ""
```

Comment:
Do empty string ("") and `undefined` have different meanings?

Reply:
```yaml
description:
iconUrl:
```
The behaviors are the same. `undefined` or `null` will result in the default values, which are empty string.

Reply:
When `iconUrl` is empty, bot framework will use its default icon.